src/pages/Profile.jsx

```diff
@@ -0,0 +1,209 @@
+    setWeight(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleBloodTypeChange = (e) => {
+    setBloodType(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleMedConditionsChange = (e) => {
+    setMedicalConditions(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleAllergiesChange = (e) => {
+    setAllergies(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handlePhoneChange = (e) => {
+    setPhone(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmailChange = (e) => {
+    setEmail(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmer1NameChange = (e) => {
+    setEmer1Name(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmer1PhoneChange = (e) => {
+    setEmer1Phone(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmer1EmailChange = (e) => {
+    setEmer1Email(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmer2NameChange = (e) => {
+    setEmer2Name(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmer2PhoneChange = (e) => {
+    setEmer2Phone(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmer2EmailChange = (e) => {
+    setEmer2Email(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleUserNameChange = (e) => {
+    setUserName(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handlePasswordChange = (e) => {
+    setPassword(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setProfileLog([...profileLog, itemData]);
+    setItemData({});
+    // navigate("/home");
+  };
+  // console.log(dataProfile)
+
+  return (
+    <div className="base-container">
+      <BackButton />
+      <p className="font-light w-screen mt-2 text-3xl text-dark-pink text-left pl-10">
+        Profile
+      </p>
+      <Form
+        name={name}
+        age={age}
+        weight={weight}
+        bloodType={bloodType}
+        medicalConditions={medicalConditions}
+        allergies={allergies}
+        phone={phone}
+        email={email}
+        emer1Name={emer1Name}
+        emer1Phone={emer1Phone}
+        emer1Email={emer1Email}
+        emer2Name={emer2Name}
+        emer2Phone={emer2Phone}
+        emer2Email={emer2Email}
+        userName={userName}
+        password={password}
```

Comment:
Reiterating on my previous point of using an object. When you look at it, and it seems overwhelming, and while writing it, it feels like a big chore - then let's refactor it into a better data structure :)! Engineers should strive to be lazy and do as little manual work as possible haha